Web/Scotty/Internal/Types.hs

```diff
@@ -23,6 +23,7 @@ import           Control.Monad.Trans.Except
 import qualified Data.ByteString as BS
 import           Data.ByteString.Lazy.Char8 (ByteString)
 import           Data.Default.Class (Default, def)
+import qualified Data.Semigroup as Sem
```

Comment:
Now that we import `Prelude.Compat`, my hope is that this import will no longer be necessary.